Source/Extensions/CAShapeLayerExtension.swift

```diff
@@ -13,18 +13,14 @@ extension CAShapeLayer {
     /// - Parameter style: Protocol that provide which edge or corner will be added with style
     func createCustomBorder(with style: ElementBorderStyle) {
 
-        // if path not nil then it's already added so no need to draw lines or curves again
-        guard self.path == nil else { return }
-
         // always fill is clear color
         fillColor = UIColor.clear.cgColor
 
         lineWidth = style.borderWidth
-        let rect = bounds.insetBy(dx: style.cornerRadius / 2,
-                                  dy: style.cornerRadius / 2)
+
         let path = UIBezierPath()
-        applyBorders(with: path, rect: rect, style: style)
-        applyCorners(with: path, rect: rect, style: style)
```

Comment:
Would like to understand what changed here and why its better to remove the offset now?